src/features/AcademicProgram.tsx

```diff
@@ -80,7 +80,7 @@ const AcademicProgram = () => {
       return;
     }
 
-    if (fields && Array.isArray(fields) && fields.length > 1) {
+    if (fields && Array.isArray(fields) && fields[1] != null) {
       fields = fields.join(', ');
```

Comment:
Rather than checking for null in the if-statement, you can handle any scenario by filtering out falsey (null, undefined) elements of the array prior to joining. The code `.filter(Boolean)` is shorthand for `.filter(item => Boolean(item))` .. both pass each item in the array to the Boolean constructor which coerces the item into truthiness or falseyness (null, undefined, and empty strings are falsey while populated strings are truthy).. so any populated string returns true which the `filter` keeps in the resulting array.

`fields = fields.filter(Boolean).join(', ');`